source/dub/package_.d

```diff
@@ -375,6 +375,10 @@ class Package {
 		return ret;
 	}
 
+	string[] getBuildTypes(){
```

Comment:
This may be a bit too specific to warrant a public method, since the external code almost as concisely directly call `.recipe.buildTypes.keys` instead. But a version of this method that provides enough own value would be one that returns a range of *all* possible build types, including the pre-defined ones. That would also move the list of pre-defined version from commandline.d closer to where it is actually defined.